pandas/core/frame.py

```diff
@@ -4945,11 +4947,12 @@ def quantile(self, q=0.5, axis=0, numeric_only=True,
         Returns
         -------
         quantiles : Series or DataFrame
-            If ``q`` is an array, a DataFrame will be returned where the
-            index is ``q``, the columns are the columns of self, and the
-            values are the quantiles.
-            If ``q`` is a float, a Series will be returned where the
-            index is the columns of self and the values are the quantiles.
+
+            - If ``q`` is an array, a DataFrame will be returned where the
```

Comment:
Yes, indeed, that's the rule for rst lists and also apply in docstrings, *unless* it's the only thing in the explanation of a parameter (no text before or after in the entry), then no blank lines are needed (eg in this case it is normally not needed)